microsoft-edge/webview2/concepts/data_privacy.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Data & Privacy in WebView2
+description: Overview of WebView2 Data Collection and Practices. 
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 05/03/2023
+---
+# Data & Privacy in WebView2 
+
+## Background
+The WebView2 team tracks data to help understand and prioritize our customers' requirements, diagnose and debug issues, and overall improve WebView2 as a product.
+Developers consent to this data collection in the WebView2 Runtime Terms and Conditions License, or in the Windows license. Additionally, we follow the standards outlined in the [Microsoft Edge Privacy Whitepaper](https://learn.microsoft.com/en-us/microsoft-edge/privacy-whitepaper/).
+
+In general, all of our data collection goes through the same strict standards as Microsoft Edge. The main telemetry sources are Chromium and Edge telemetry infrastructure, Windows data reporting, and Watson (Microsoft Internal) infrastructure to collect crash dumps.
+With any data the WebView2 team collects, we comply/conform with the laws and regulations about best privacy practices, and have guardrails to protect against privacy issues.
+
+## Optional Diagnostic Data
+
+WebView2 and Edge respect the Windows Optional Diagnostic Data toggle, so end users have control over any optional data WebView2 collects. This toggle can only be modified by the end user. The only mandatory data that is collected is the [required events](https://learn.microsoft.com/en-us/windows/privacy/required-windows-11-diagnostic-events-and-fields) which are required from any app running on Windows.
+
+WebView2 is considered a Windows component, and the data collection consent is governed by Windows Diagnostic data setting on Windows 10 & 11 as a centralized switch.
+End users are empowered to control the data collection of WebView2 and can do so via toggling the Windows Diagnostic data setting on Windows 10 & 11. On Windows 7/8.1, given there is no Windows Diagnostic data setting, we treat this as no consent for optional data and thus optional diagnostic data is not collected. There is very limited required data that the OS always collects, unless you're on some specific SKUs. Developers are welcome to convey this to their end users and ask them to use the Windows Diagnostic data setting to fit the end user’s preference.
+Developers currently have no control over this Windows Diagnostic data setting, similar to the Windows APIs developers use.
+

```

Comment:
maybe add screenshot, could help wording in article.  Is this what this article means by "the **Windows Diagnostic data** setting"?

like:
![image](https://user-images.githubusercontent.com/45407486/236305737-8fc3c98b-cb6f-4c4e-88a2-87cd242d3808.png)

Reply:
I'm going to change the wording to **Diagnostic data** setting - this is what I see in this article - https://support.microsoft.com/en-us/windows/diagnostics-feedback-and-privacy-in-windows-28808a2b-a31b-dd73-dcd3-4559a5199319.

Reply:
Also, the diagnostic data setting looks different on different platforms so I will update the first reference to Settings > Privacy & security > Diagnostics & feedback > Diagnostic data

Reply:
The above screenshot shows which Windows setting/control I think we're discussing in-depth.  Per industry Docs conventions, it would be good to provide the specific UI control label on Windows and each platform and not leave that to guessing and leave it implicit.  Can we give the specific UI control name on each platform?  Because the article focuses on the specific setting, so we should accurately identify which specific setting we're discussing in-depth.

Reply:
When you say specific UI control name, does that mean the full path to that setting? Also, does this mean needing to add the screenshot of the setting?

Reply:
maybe this comment thread can be Resolved

Reply:
https://support.microsoft.com/en-us/windows/diagnostics-feedback-and-privacy-in-windows-28808a2b-a31b-dd73-dcd3-4559a5199319
never names the control, it just says "the option":  
![image](https://github.com/MicrosoftDocs/edge-developer/assets/45407486/d09d943b-5a58-4938-b37c-745b786c01ff)

Reply:
Ok, I will keep this as is and resolve this comment, feel free to re-open thread.